.storybook/preview.tsx

```diff
@@ -2,6 +2,7 @@ import React from "react";
 import { MockedProvider } from "@apollo/client/testing";
 import { createMemoryRouter, RouterProvider } from "react-router-dom";
 import { Parameters, Decorator } from "@storybook/react";
+import { WithApolloClient } from "storybook-addon-apollo-client/dist/decorators";
```

Comment:
Do you think this applies? ([source](https://github.com/lifeiscontent/storybook-addon-apollo-client))

<img width="827" alt="Screenshot 2023-07-06 at 4 40 06 PM" src="https://github.com/evergreen-ci/spruce/assets/47064971/d941db8d-26c7-4dec-9c70-a12764b74c18">

Reply:
It's still required to get it to work with storyshots. https://github.com/lifeiscontent/storybook-addon-apollo-client/issues/16

Reply:
oh I see, can we leave a comment about this in the code so it doesn't get lost?